src/helpers/plainAddPlaceholder/getPagesDictionaryRef.js

```diff
@@ -0,0 +1,19 @@
+import SignPdfError from '../../SignPdfError';
+
+const getPagesDictionaryRef = (info) => {
+    const catalogPosition = info.root.toString().indexOf('/Type/Catalog');
+
+    if (catalogPosition < 0) {
+        throw new SignPdfError(
+            'Failed to find the pages descriptor. This is probably a problem in node-signpdf.',
+            SignPdfError.TYPE_PARSE,
+        );
+    }
```

Comment:
Is this check necesary? If the pdf doesn't have a catalag it should fail elsewhere, am I wrong?

Reply:
This PR is somewhat... wrong... Looking at all those commits... But, yes, seems like this check is not where it should be.

Reply:
I think you see a lot of commits beacause this modification was based on the play-with-trailers branch.

Reply:
I've made a new PR based on your work and the develop branch of this repo https://github.com/vbuch/node-signpdf/pull/38

Reply:
Hi, I have been reviewing the project, I see that in the "play-with-trailers" branch there is a pull request for multiple signatures, I got a bit confused, first of all I want to ask if this branch is based on:
1. Sign a document several times, or,
2. Adapt for any type of signature to work.
Remembering that there are several types of signatures. If I am correct (1), I am very interested in the fusion of this branch to be able to work on it, of course, if you need help, could you tell me where and the objective to solve it?

Reply:
@theprobinski Yes, you can sign a document several times, with multiple different signatures (but the cert type has to be pkcs12). Maybe you should check out the new PR created by @andres-blanco.

Reply:
The play-with-trailers branch was merged 3 months ago. I don't have the time to review the incoming changes right now so I can hope the community will do that. Check the PR made by @andres-blanco. It should be having the same changes as this one but rebased.

Reply:
It is very interesting, I have been reviewing and running the tests of the PR made by @andres-blanco, I have tried with the file "w3dummy.pdf" and the tests run well, but when using another PDF, for example the same file which is in the same resources called "contributing.pdf", goes through an error where, I understand that it searches the buffer "type/Catalog" and does not find it since it has a space between the "/". Why do you think these differences exist between the pdfs?
Attached the error capture, I put a console.log to print that was in the pdf "w3dummy.pdf", and the "contributing.pdf" to find the difference.
I hope you can get me out of doubt, thank you very much.
![otherFile-vs-w3dummy](https://user-images.githubusercontent.com/57117401/67800081-ad1bfc00-fa54-11e9-99be-88ad1128e7b9.png)

Reply:
As I said, this PR is somewhat messed up. Here is how this is done in master and develop (which are even at the moment): https://github.com/vbuch/node-signpdf/blob/develop/src/helpers/plainAddPlaceholder/getPagesDictionaryRef.js#L7